li-hadoop-plugin/src/main/groovy/com/linkedin/gradle/liazkaban/LiAzkabanPlugin.groovy

```diff
@@ -56,4 +60,25 @@ class LiAzkabanPlugin extends AzkabanPlugin {
   AzkabanDslCompiler makeCompiler(Project project) {
     return new LiAzkabanDslCompiler(project);
   }
+
+  /**
+   * Prints Linkedin specific message after upload to Azkaban completes, mostly to notify users of
+   * upcoming changes which affects Azkaban users. Subclasses can override this method.
+   */
+  @Override
+  void printUploadMessage() {
+    try {
+      String message = MESSAGE_URL.toURL().getText();
+      if (!message.isEmpty() && message.length() > 0) {
+        logger.lifecycle("\n"+message);
+      }
+    } catch (HttpException ex) {
```

Comment:
Perhaps we can just catch Exception here. Then we can remove the dependency on HttpException in this class. Also, I don't think we should print the stacktrace here - that could be annoying for users. Instead, you might print e.getMessage() as a part of the logger.error message.